debian/7.2/Dockerfile

```diff
@@ -1,36 +1,38 @@
 # syntax = docker/dockerfile:1.0.2-experimental
-ARG SPRYKER_PHP_VERSION=7.2.24
+ARG SPRYKER_PHP_VERSION=7.2.28
 
-FROM php:${SPRYKER_PHP_VERSION}-fpm-alpine3.9
+FROM php:${SPRYKER_PHP_VERSION}-fpm
```

Comment:
This suggestion applies to other places as well:
I would rather suggest that you stick to specific versions (like `-alpine3.9` or `-alpine3.10`. You refer to specific alpine version in several places in this file (ie. in readme). If at some time upstream `-alpine` tag switches to 3.11, this will indirectly cause your further builds to switch to alpine 3.11 as well, without you even knowing that. Such surprises aren't welcome, it's better to explicitly specify versions

Reply:
Good point.